README.md

```diff
@@ -128,7 +170,10 @@ We provide training script for both text-to-video and image-to-video generation
   MAX_TRAIN_STEPS=("3000")
   ```
 
-- Select which Accelerate configuration you would like to train with: `ACCELERATE_CONFIG_FILE="accelerate_configs/uncompiled_1.yaml"`. We provide some default configurations in the `accelerate_configs/` directory - single GPU uncompiled/compiled, 2x GPU DDP, DeepSpeed, etc. You can create your own config files with custom settings using `accelerate config --config_file my_config.yaml`.
+- Select which Accelerate configuration you would like to train
+  with: `ACCELERATE_CONFIG_FILE="accelerate_configs/uncompiled_1.yaml"`. We provide some default configurations in
+  the `accelerate_configs/` directory - single GPU uncompiled/compiled, 2x GPU DDP, DeepSpeed, etc. You can create your
+  own config files with custom settings using `accelerate config --config_file my_config.yaml`.
```

Comment:
Is this because of your PyCharm formatting?

Reply:
@zRzRzRzRzRzRzR Could you revert the PyCharm introduced formatting, please? I think there should also be a setting to disable it. It would be nice to not have line wrapping within the README itself because GitHub view automatically handles this